### ISSUE TITLE
alt text for image should be not empty and more descripitive

### ISSUE THREAD

**Issue by AhmedAdelSelim** [MahmoudHashemSE/Udemy-Clone#3]
1- for example we should use " alt=" clock " instead of alt="Time" because it is more descriptive
